src/Sarif.Multitool/Rules/RuleResources.resx

```diff
@@ -180,4 +180,10 @@
   <data name="SARIF1014_UriBaseIdRequiresRelativeUri" xml:space="preserve">
     <value>If a fileLocation object contains a "uriBaseId" property, the value of the "uri" property must be a relative URI reference.</value>
   </data>
+  <data name="SARIF1015_Default" xml:space="preserve">
+    <value>{0}: The SARIF specification requires the value of this property to be an absolute URI, but "{1}" is a relative URI reference.</value>
```

Comment:
>SARIF specification requires the value [](start = 20, length = 38)

the SARIF spec requires all of this validation, so it is not helpful to note this. instead, prefer the spec language itself, such as The value of this property must be an absolute URI, or should or whatever.

Reply:
How's this: "The value of this property is required to be an absolute URI, but "{1}" is a relative URI reference."

---
In reply to: [211695585](https://github.com/Microsoft/sarif-sdk/pull/1040#discussion_r211695585) [](ancestors = 211695585)